### DISCUSSION TITLE
Test Results HTML not Loading

### DISCUSSION THREAD

**Discussion by jagsandhu** [maester365/maester#595] [General]
Has anyone come across an issue, where ran against a fairly large environment the test-results html file loads up blank, despite being over 5mb is size and the md file being 375mb.  Using DevTools, I cannot see anything obvious, other than the message:

TypeError: Cannot read properties of null (reading 'map')
    at ResultInfoDialog
    
 Anyone able to give an pointers?
 Thanks

**Comment by soulemike** [comment]
I haven't heard of issues in a few environments in the 1,000s of users space. What is the scale you are noticing this in?

**Comment by mrhousz** [comment]
I am experiencing this issue. html file doesn't load, dev tools has two errors: 

TestResults-2025-01-06-213813.html:4276 TypeError: Cannot read properties of null (reading 'toLowerCase')

This is the function with the error:

`function n$b(r2, o3) {
  return React.Children.map(o3, (e4) => {
    var o4;
    if ((null !== (o4 = t$1(e4)) && void 0 !== o4 ? o4 : e4.props.value).toLowerCase().includes(r2.toLowerCase()))
      return e4;
  });
}`

TypeError: Cannot read properties of null (reading 'map')

**Comment by jcossin03** [comment]
I am also experiencing this issue. Environment is not large though (1600~) and md file is not gigantic like OP mentions. File will only show blank white screen when attempting to open from edge / chrome. Sending to IOS and opening in edge just displays the code in browser. Testresults file is 4300kb.

**Comment by merill** [comment]
@jagsandhu @jcossin03 @mrhousz Are you still seeing this issue?

Can you please test with the new module in preview (using Install-Module Maester -AllowPreRelease)?

If you are still seeing this, any chance you can share the json file? Feel free to take out all tenant details and email them to.

**Reply by jcossin03** [reply]
Confirmed working for me this morning. TY

**Reply by withLogic** [reply]
I can confirm that I was also having this issue and the -AllowPreRelease flag has fixed it. Thank you!

**Reply by jagsandhu** [reply]
Confirmed working for me too. Many thanks :-)

**Reply by evolstuer** [reply]
Also worked for me
`Uninstall-Module Maester`
`Install-Module Maester -AllowPreRelease -Force`

**Comment by merill** [comment]
This is now in the production v1.1
